frontend/lang/italiano.json

```diff
@@ -0,0 +1,311 @@
+    "t_process_give_permission_to_run_code": "Esegui codice del notebook",
+    "t_process_running_cells": "Esecuzione celle... ({{done}}/{{total}} completate)",
+    "t_safe_preview": "Anteprima sicura",
+    "t_safe_preview_not_executed": "Codice non eseguito in <em>Anteprima sicura</em>",
+    "t_safe_preview_not_rendered": "Script e stili non renderizzati in <em>Anteprima sicura</em>",
+    "t_safe_preview_confirm_warning": "Avviso",
+    "t_safe_preview_confirm_before": "Sei sicuro di fidarti di questo file?",
+    "t_safe_preview_confirm_before_danger": "⚠️ Pericolo!",
+    "t_safe_preview_confirm_after": "Un notebook dannoso può rubare password e dati.",
+    "t_safe_preview_body": "Stai leggendo e modificando questo file senza eseguire il suo codice Julia.",
+    "t_safe_preview_run_this_notebook": "Quando sei pronto, puoi {{run_this_notebook}}.",
+    "t_safe_preview_run_this_notebook_link": "eseguire questo notebook",
+    "t_instant_feedback_ellipsis": "Feedback istantaneo...",
+    "t_instant_feedback_send": "Invia",
+    "t_how_can_we_improve": "🙋 Come possiamo migliorare {{pluto}}?",
+    "t_FAQ": "FAQ",
+
+    "t_loading_cells": "Caricamento celle...",
+    "t_reading_hidden_code": "👀 Lettura codice nascosto",
+    "t_enable_and_run_cell": "Abilita ed esegui la cella",
+    "t_disable_this_cell_and_all_cells_that_depend_on_it": "Disabilita questa cella e tutte le celle che dipendono da essa",
+    "t_enable_cell_action": "Abilita cella",
+    "t_disable_cell_action": "Disabilita cella",
+    "t_delete_cell_action": "Elimina cella",
+    "t_show_logs_action": "Mostra log",
+    "t_hide_logs_action": "Nascondi log",
+    "t_show_logs_action_description": "Mostra i log della cella",
+    "t_hide_logs_action_description": "Nascondi i log della cella",
+    "t_copy_output_action": "Copia output",
+    "t_copy_output_action_description": "Copia l'output di questa cella negli appunti (clipboard).",
+    "t_enable_in_file_action": "Abilita nel file",
+    "t_enable_in_file_action_description": "Questa cella è attualmente memorizzata nel file del notebook come commento Julia. Clicca qui per abilitarla.",
+    "t_disable_in_file_action": "Disabilita nel file",
+    "t_disable_in_file_action_description": "Memorizza questo codice nel file del notebook come commento Julia. In questo modo, non verrà eseguito quando il notebook viene eseguito come script al di fuori di Pluto.",
+    "t_interrupt_cell": "Interrompi (Ctrl + Q)",
+    "t_save_cell": "Salva codice senza eseguire",
+    "t_jump_cell": "Questa cella dipende da una cella disabilitata",
+    "t_run_cell": "Esegui cella (Shift + Enter)",
+    "t_help_about_tab_navigation_inside_cell": "Premi <kbd>Esc</kbd> e poi <kbd>Tab</kbd> per continuare la navigazione.",
+
+    "t_cell_is_disabled": "Questa cella è disabilitata. {{link}} per eseguire il codice.",
+    "t_cell_is_disabled_link": "Abilita questa cella",
+    "t_ask_ai_action": "Chiedi all'AI",
```

Comment:
Honestly I'm not sure whether this should be AI (Artificial Intelligence, in English) or IA (Intelligenza Artificiale, in Italian)